test/column/formatter.test.tsx

```diff
@@ -110,3 +110,37 @@ describe('Custom formatter component', () => {
     });
   });
 });
+
+test.skip('Cell should not steal focus when the focus is outside the grid and cell is recreated', async () => {
```

Comment:
FWIW we can also do this, it'll run the test.
```suggestion
test.failing('Cell should not steal focus when the focus is outside the grid and cell is recreated', async () => {
```


If we ever accidentally fix this test, it'll make jest log this:
![image](https://user-images.githubusercontent.com/567105/233453926-bee21acf-04c2-4da7-813d-8c264b20d3dc.png)
and so we could remove `.failing` to ensure the test keeps passing and isn't ignored.

Reply:
Nice 👍